src/lib/Footer.svelte

```diff
@@ -0,0 +1,57 @@
+<footer>
+  <hr />
+  <div class="footer-content">
+    <div>
+      <a href="https://bsaber.com">
+        <img class="logo" alt="BeastSaber" src="/BeastSaber-LogoW.webp" /></a
+      >
+    </div>
+    <div class="links">
+      <p>
+        <a class="btn" href="https://discord.gg/VJZHUbt" target="_blank">Discord</a> <br />
```

Comment:
I've always used `<br />` but that might not be the best method. Are there any adverse effects to the `<br />`?

For what it's worth, I tried to style it with `<ul>` and it was causing formatting issues. I was probably doing something wrong though.

Reply:
The differences to the user are quite small. They select differently though. Rule of thumb I apply is: Do I wanna select these together (like multiple address lines) -> `<br />`. Otherwise, `<ul>` and `<li>` (or similar).

If you try these you should notice that the selection doesn't wrap as "fast". There is a bit of a resistance.

It's quite minor - I don't think it's too important, though. I'm not sure if there are other implications. 

You can try this if you wanted to go with it:

```html
    <div class="links">
      <ul>
        <li><a class="btn" href="https://discord.gg/VJZHUbt" target="_blank">Discord</a></li>
        <li>
          <a
            class="btn"
            href="https://patreon.com/user?u=119426555&utm_medium=unknown&utm_source=join_link&utm_campaign=creatorshare_creator&utm_content=copyLink"
            target="_blank">Patreon</a
          >
        </li>
        <li>
          <a class="btn" href="https://youtube.com/@BeastSaberNews" target="_blank">YouTube</a>
        </li>
        <li><a class="btn" href="https://x.com/BeastSaberNews" target="_blank">Twitter</a></li>
      </ul>
    </div>
```

```css
  ul {
    list-style-type: none;
    padding: 0;
    margin: 0;
  }
```

Reply:
This looked good to me so I went with it instead. I didn't see any adverse effects here.

Thanks for the code suggestion!